diff --git a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/PromiseImpl.kt b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/PromiseImpl.kt
index c0a975c..8184c2c 100644
--- a/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/PromiseImpl.kt
+++ b/node_modules/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/PromiseImpl.kt
@@ -1,11 +1,8 @@
 package expo.modules.kotlin.jni
 
-import expo.modules.BuildConfig
 import expo.modules.core.interfaces.DoNotStrip
 import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.Promise
-import expo.modules.kotlin.exception.PromiseAlreadySettledException
-import expo.modules.kotlin.logger
 import java.lang.ref.WeakReference
 
 @DoNotStrip
@@ -61,17 +58,7 @@ class PromiseImpl @DoNotStrip internal constructor(
 
   private inline fun checkIfWasSettled(body: () -> Unit) {
     if (wasSettled) {
-      val exception = PromiseAlreadySettledException(fullFunctionName ?: "unknown")
-      val jsLogger = appContextHolder?.get()?.jsLogger
-      // We want to report that a promise was settled twice in the development build.
-      // However, in production, the app should crash.
-      if (BuildConfig.DEBUG && jsLogger != null) {
-        jsLogger.error("Trying to resolve promise that was already settled", exception)
-        logger.error("Trying to resolve promise that was already settled", exception)
-        return
-      }
-
-      throw exception
+      return
     }
 
     body()
