diff --git a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
index eba25ec..f18dc61 100644
--- a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
+++ b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
@@ -117,8 +117,8 @@ function createAnimatedComponent(Component) {
     animatedCache.set(Component, AnimatedComponent);
     return AnimatedComponent;
 }
-function printUpgradeWarning(warning, originalProps) {
-    console.log(`CssInterop upgrade warning.\n\n${warning}.\n\nThis warning was caused by a component with the props:\n${stringify(originalProps)}\n\nIf adding or removing sibling components caused this warning you should add a unique "key" prop to your components. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key\n`);
+function printUpgradeWarning(warning, _originalProps) {
+    console.log(`CssInterop upgrade warning.\n\n${warning}.\n\nIf adding or removing sibling components caused this warning you should add a unique "key" prop to your components. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key\n`);
 }
 function stringify(object) {
     const seen = new WeakSet();
diff --git a/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx b/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
index cfb9a9e..5fc0375 100644
--- a/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
+++ b/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
@@ -227,10 +227,10 @@ function createAnimatedComponent(Component: ComponentType<any>): any {
 
 function printUpgradeWarning(
   warning: string,
-  originalProps: Record<string, any> | null | undefined,
+  _originalProps: Record<string, any> | null | undefined,
 ) {
   console.log(
-    `CssInterop upgrade warning.\n\n${warning}.\n\nThis warning was caused by a component with the props:\n${stringify(originalProps)}\n\nIf adding or removing sibling components caused this warning you should add a unique "key" prop to your components. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key\n`,
+    `CssInterop upgrade warning.\n\n${warning}.\n\nIf adding or removing sibling components caused this warning you should add a unique "key" prop to your components. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key\n`,
   );
 }
 
